snuba/core/initialize.py

```diff
@@ -29,8 +29,8 @@ def _load_entities() -> None:
     initialize_entity_factory()
 
 
-def initialize() -> None:
-    logger.info("Initializing snuba")
+def initialize_snuba() -> None:
```

Comment:
Simple rename to make it easier to understand the usage of this function outside of this module.